content/en/blog/_posts/2025-04-23-jobs-successpolicy-goes-ga.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: blog
+title: "Kubernetes 1.33: Job's SuccessPolicy Goes GA"
+date: 2025-04-23
+slug: kubernetes-1-33-jobs-success-policy-goes-ga
+authors: >
+  [Yuki Iwai](https://github.com/tenzen-y) (CyberAgent, Inc)
+---
+
+Announcing the graduation to General Availability (GA) of the Job's __Success Policy__, in Kubernetes v1.33.
+
+## About Job's Success Policy
+
+When you run batch workloads with leader-worker patterns, you might want to mark it as succeeded
+even if some index failed. On the other hand, the Kubernetes Job without __Success Policy__ requires all Pods succeeded for Job succeeded.
+
+This Job's __Success Policy__ allows you to specify what is criteria for Job succeeded under the `.spec.successPolicy` 
+opposed to the default success criteria which is all indexes succeeded.
```

Comment:
```suggestion
Job's __Success Policy__ allows you to specify the early exit criteria for a Job using  the `.spec.successPolicy` 
field. Which describes a set of rules either using a list of succeeded indexes for a job, or defining a minimal required size of succeeded indexes. 
```

Reply:
That makes sense since your recommendation describes actual new fields, thanks

Reply:
> It might be good to preface this with a statement that this feature is only available for indexed jobs.

That makes sense.